modules/preview-react/form-field/lib/FormField.tsx

```diff
@@ -19,7 +19,6 @@ export interface FormFieldProps extends Omit<ExtractProps<typeof Stack, never>,
   orientation: 'vertical' | 'horizontal';
   /**
    * Spacing between children elements
-   * @default xxxs when vertical l when orientation is horizontal
```

Comment:
I'm not sure why we removed this. I don't know where it renders besides in the editor. Seems unnecessary to take it out.

It could certainly be improved:
```
* @default xxxs when orientation is vertical, l when orientation is horizontal
```

or moved into the line above it:
```
* Spacing between children element. The default value is xxxs when orientation is vertical, l when orientation is horizontal.

Reply:
This is why: 
<img width="836" alt="image" src="https://user-images.githubusercontent.com/338257/221289111-1ef4e3b6-512c-4e45-b835-326396049e69.png">

It isn't really a default at all because it is conditional based on other criteria.

I've made the component parser smarter to pick up more defaults without the default being in the JSDoc where they can get out of sync.

For example, here's supported scenarios where default will be picked up:

```tsx
// in destructured assignment
({
  gap = "s"
}: Props) => {
}

// in JSX
return <Flex gap="s" {...elemProps}>
```

I can add to the description instead.